lang/go/eval/symbol/series.go

```diff
@@ -28,8 +30,32 @@ type SeriesSymbol struct {
 	Elem  Symbols     `ko:"name=elem"`
 }
 
-var _ Symbol = &SeriesSymbol{}
+var (
+	_            Symbol = &SeriesSymbol{}
+	seriesGoType        = reflect.TypeOf([]interface{}{})
+)
+
+// DisassembleToGo converts a Ko value into a Go value
+func (ss *SeriesSymbol) DisassembleToGo(span *model.Span) (reflect.Value, error) {
```

Comment:
In general, it is better to preserve as much type info as you can:
Here, instead of returning a []interface{}, you could return []T, where
T is the actual type in the slice. 

Preserving type information accurately is handy when you disassemble a Ko value into Go and then you assemble it back in. You want to get the same result. 

Since preserving type information is not crucial for your application. I wouldn't bother implementing it now. Just put a comment that this should be done eventually.